utils.go

```diff
@@ -724,3 +726,34 @@ func (ae *atomicError) Value() error {
 	}
 	return nil
 }
+
+func cloneTLSConfig(c *tls.Config) *tls.Config {
```

Comment:
This was also added for backwards-compatibility. Remove the function and use `c.Clone` directly instead.

Reply:
I did 6649058